src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -178,9 +185,20 @@ export class MatAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
               @Optional() @Host() private _formField: MatFormField,
               @Optional() @Inject(DOCUMENT) private _document: any,
               // @deletion-target 7.0.0 Make `_viewportRuler` required.
-              private _viewportRuler?: ViewportRuler) {}
+              private _viewportRuler?: ViewportRuler) {
+
+    if (typeof window !== 'undefined') {
+      _zone.runOutsideAngular(() => {
+        window.addEventListener('blur', this._windowBlurHandler);
+      });
+    }
+  }
 
   ngOnDestroy() {
+    if (typeof window !== 'undefined') {
+      window.removeEventListener('blur', this._windowBlurHandler);
```

Comment:
This will be a good use for the [Page Lifecycle API](https://developers.google.com/web/updates/2018/07/page-lifecycle-api)... someday.

Reply:
The browser support isn't too horrible tbh, but I think that this is case is simple enough that we can get away with the event.